app/tasks/mail.js

```diff
@@ -0,0 +1,34 @@
+const nodemailer = require('nodemailer');
+const createWelcomeMail = require('../email_templates/welcome.js');
+const dotenv = require('dotenv');
+dotenv.config();
+
+module.exports = async (payload, helpers) => {
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.mailtrap.io',
+    port: 2525,
+    auth: {
+      user: process.env.MAIL_IO_USER,
+      pass: process.env.MAIL_IO_PASS
+    }
+  });
```

Comment:
is there a way we can consolidate this into a single connection string? (new issue, new PR)